charts/enterprise-logs/values.yaml

```diff
@@ -0,0 +1,404 @@
+# The default values specified in this file are enough to deploy all of the
+# Grafana Enterprise Logs microservices but are not suitable for production
+# load.
+# To configure the resources for production load, refer to the the small.yaml or
+# large.yaml values files.
+
+# Container image settings.
+# Since the image is unique for all microservices, so are image settings.
+image:
+  repository: grafana/enterprise-logs
+  tag: v1.1.0
+  pullPolicy: IfNotPresent
+  # Optionally specify an array of imagePullSecrets.
+  # Secrets must be manually created in the namespace.
+  # ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - /
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+serviceAccount:
+  create: true
+  name:
+  annotations: {}
+
+useExternalConfig: false
+externalConfigSecretName: 'enterprise-logs-config'
+externalConfigVersion: '0'
+
+# In order to use Grafana Enterprise Logs features, you will need to provide the contents of your Grafana Enterprise Logs
+# license, either by providing the contents of the license.jwt, or the name Kubernetes Secret that contains your license.jwt.
+# To set the license contents, use the flag `--set-file 'license.contents=./license.jwt'`
+# To use your own Kubernetes Secret, `--set license.external=true`.
+useExternalLicense: false
+license:
+  contents: "NOTAVALIDLICENSE"
+  external: false
+  secretName: 'enterprise-logs-license'
+
+tokengenJob:
+  enable: true
+  extraArgs: {}
+  env: []
+  annotations: {}
+
+config:
```

Comment:
Aren't we looking to orient around a single configuration string?

Reply:
I see that you noted this in your PR description, let's discuss this in our meeting today and follow up here.